### PR TITLE
#230 - Small refactoring

### PR DESCRIPTION
**Pull request by sasha240100**
**This PR is related to issue #230**

### CODE REVIEW

**Comment by hirako2000** [comment]
Is build passing?

File: examples/assets/css/style.css [hunk 1]
```diff
@@ -35,13 +35,8 @@ div.ui.sidebar.vertical.inverted.menu.left.overlay.visible {
   background: #262C38;
   margin-bottom: 0;
 }
-.ui.header > .image:not(.icon), .ui.header > img {
```
Comment (hirako2000) [inline examples/assets/css/style.css]: was this intended?

File: examples/basic/fogexp/script.js [hunk 2]
```diff
@@ -10,17 +10,21 @@ const cameraModule = new WHS.app.CameraModule({
...
 const world = new WHS.App([
-  ...UTILS.appModules({
-    position: new THREE.Vector3(0, 10, 200)
+  new WHS.ElementModule(),
+  new WHS.SceneModule(),
+  cameraModule,
+  new WHS.RenderingModule(UTILS.appDefaults.rendering, {
+    shadow: true
   }),
+  new PHYSICS.WorldModule(UTILS.appDefaults.physics),
```
Comment (hirako2000) [inline examples/basic/fogexp/script.js]: we don't use physics in this example.

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "version": "gulp build --prod && git add .",
     "test": "xo ./src/**/*.js && jest",
-    "start": "webpack-dashboard -t WhitestormJS -- gulp dev --devPhysics 8001",
```
Comment (hirako2000) [inline package.json]: is that intentional?

File: examples/assets/css/style.css (hunk 1 above)
Comment (sasha240100) [inline examples/assets/css/style.css]: @hirako2000 Seems like automatically...

File: examples/basic/fogexp/script.js (hunk 2 above)
Comment (sasha240100) [inline examples/basic/fogexp/script.js]: @hirako2000 Thanks! Going to fix that

File: package.json (hunk 3 above)
Comment (sasha240100) [inline package.json]: @hirako2000 Yes, we don't need it in whs now

**Comment by hirako2000** [comment]
the doc is updated with those changes? We kept the old stuff to keep Beta compatible right?

**Comment by sasha240100** [comment]
@hirako2000 I think no need to keep alias for `Model`. We will make alias once release a stable version

**Comment by sasha240100** [comment]
@thejmazz @utrobin Waiting for your review

**Pull request merged**